Fix:

error: call to undeclared library function 'exit' with type 'void (int) __attribute__((noreturn))'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
error: call to undeclared function 'filedlg_show'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]

https://sourceforge.net/p/aewan/bugs/16/
--- configure
+++ configure
@@ -2581,6 +2581,7 @@
 cat >>conftest.$ac_ext <<_ACEOF
 /* end confdefs.h.  */
 #include <ctype.h>
+#include <stdlib.h>
 #if ((' ' & 0x0FF) == 0x020)
 # define ISLOWER(c) ('a' <= (c) && (c) <= 'z')
 # define TOUPPER(c) (ISLOWER(c) ? 'A' + ((c) - 'a') : (c))
--- debug_aewl.c
+++ debug_aewl.c
@@ -7,6 +7,7 @@
 #include "aewl/field.h"
 #include "aewl/label.h"
 
+#include "filedlg.h"
 #include "keys.h"
 
 #include <ncurses.h>
